backend/typescript/services/implementations/camperService.ts

```diff
@@ -97,10 +103,12 @@ class CamperService implements ICamperService {
         {},
         { session },
       );
+
       // Ensure all session ids passed in are found in the db
       if (sessionsToRegister.length !== campSessions.length) {
         throw new Error("Not all session ids passed in were found");
       }
+
```

Comment:
I think I forgot to check if the camp is a published camp which is not completed. Could you add the camp status check (there should be a get status function somewhere in utils- also idk why a camp doesn't just have a status but here we are)